### PR TITLE
Allow to enable Azure Policy

### PR DESCRIPTION
**Pull request by mcanevet**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -87,5 +91,3 @@ resource "azurerm_log_analytics_solution" "main" {
     product   = "OMSGallery/ContainerInsights"
   }
 }
-
-
```
Comment (yupwei68) [inline main.tf:91]: Could we leave these blank lines not changed?

File: variables.tf [hunk 2]
```diff
@@ -84,4 +84,10 @@ variable "enable_http_application_routing" {
   description = "Enable HTTP Application Routing Addon (forces recreation)."
   type        = bool
   default     = false
-}
\ No newline at end of file
+}
```
Comment (yupwei68) [inline variables.tf]: Could we leave these not changed?

**Review by yupwei68 — COMMENTED** [review]
Hi @mcanevet , thanks for opening this PR. Overall, it looks fine to me. 
1. Please make the block "dynamic", set the block when "azure_policy" = true, because this feature is not enabled in Azure China or Azure US Government.
2. Could you please add this field in fixture/test/main.tf, and test it when "azure_policy" is enabled. 
3. Please add this in the example in README.md.

File: main.tf (hunk 1 above)
Comment (mcanevet) [reply main.tf]: Done (my editor automatically remove trailing new lines)

File: variables.tf (hunk 2 above)
Comment (mcanevet) [reply variables.tf]: Done (my editor automatically append new line at end of file)

**Comment by mcanevet** [comment]
@yupwei68 done

**Review by yupwei68 — APPROVED** [review]
LGTM!

**Pull request merged**